resources/language/resource.language.pt_br/strings.po

```diff
@@ -84,6 +84,14 @@ msgctxt "#30014"
 msgid "Account"
 msgstr "Conta"
 
+msgctxt "#30015"
+msgid "Other options"
+msgstr "Outras opçÕes"
```

Comment:
only one question
Is this okay with Õ doesn't it go lowercase with õ?

Reply:
Nope, it's a typo, correct is
Outras opções